### ISSUE TITLE
Bug about filter sentences

### ISSUE THREAD

**Issue by lmatt-bit** [rsennrich/Bleualign#21]
If input file include multiple articles with ".EOA" sep, the filterthreshold is not working as expected.  We need to change below code accordingly, so that correct number of lines will be filtered.

```
def filter_sentence_pairs(self, sentscores, sources_output, targets_output):
        before = 0
```

to:

```
def filter_sentence_pairs(self, sentscores, sources_output, targets_output):
        before = len(self.sources_out)
```

**Comment by lmatt-bit** [comment]
Created a pull request #22 to resolve the issue.

**Comment by rsennrich** [comment]
fixed in pull request #22
